sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -146,5 +149,74 @@ public async Task RevokeTokenWithNullUserShouldThrow()
             }
             Assert.Fail("RevokeTokensAsync should have thrown an exception.");
         }
+
+        [Test]
+        public async Task ExchangeAccessTokenWithValidToken()
+        {
+            if (TestEnvironment.ShouldIgnoreIdentityExchangeTokenTest) {
+                Assert.Ignore("Ignore exchange access token test if flag is enabled.");
+            }
+
+            string token;
+            if (Mode == RecordedTestMode.Playback)
+            {
+                token = "Sanitized";
+            } else {
+                IPublicClientApplication publicClientApplication = PublicClientApplicationBuilder.Create(TestEnvironment.CommunicationM365AppId)
+                                                    .WithAuthority(TestEnvironment.CommunicationM365AadAuthority + "/" + TestEnvironment.CommunicationM365AadTenant)
+                                                    .WithRedirectUri(TestEnvironment.CommunicationM365RedirectUri)
+                                                    .Build();
+                string[] scopes = { TestEnvironment.CommunicationM365Scope };
+                SecureString communicationMsalPassword = new SecureString();
+                foreach (char c in TestEnvironment.CommunicationMsalPassword)
+                {
+                    communicationMsalPassword.AppendChar(c);
+                }
+                AuthenticationResult result = await publicClientApplication.AcquireTokenByUsernamePassword(
+                    scopes,
+                    TestEnvironment.CommunicationMsalUsername,
+                    communicationMsalPassword).ExecuteAsync(CancellationToken.None).ConfigureAwait(false);
+                token = result.AccessToken;
+            }
+
+            CommunicationIdentityClient client = CreateClientWithConnectionString();
+            Response<AccessToken> tokenResponse = await client.ExchangeAccessTokenAsync(token);
+            Assert.IsNotNull(tokenResponse.Value);
+            Assert.IsFalse(string.IsNullOrWhiteSpace(tokenResponse.Value.Token));
+        }
+
+        [Test]
+        public async Task ExchangeAccessTokenWithEmptyTokenShouldThrow()
```

Comment:
The token could be valid but expired. Is there a way to be able to test that?

Reply:
Yeah, we should be able to generate one and add it to key vault. I'll add this case in a follow up PR. Also I added another testcase for using the operation with a dummy token.